module3/src/controllers/user.controller.ts

```diff
@@ -0,0 +1,84 @@
+import express, { Router } from 'express';
+import { Error } from 'sequelize';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import UserService from '../services/user.service.ts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { validateSchema } from '../utils/utils.ts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { userSchema } from '../validators/user.ts';
+
+const userService = UserService();
```

Comment:
If you want to write in oop style please use classes and e.g. But it is so strange to call function this way

Reply:
Moved to UserService class